test/built-ins/RegExp/prototype/exec/regexp-builtin-exec-v-flag.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 Tan Meng. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-regexpbuiltinexec
+description: RegExpBuiltinExec behavior with 'v' flag
+features: [regexp-v-flag]
+---*/
+
+const text = '𠮷a𠮷b𠮷';
+
+function doExec(regex) {
+  const result = regex.exec(text);
+  return result ? [result[0], result.index] : null;
+}
+
+assert.sameValue(doExec(/𠮷/).toString(), "𠮷,0", "Basic exec without v flag");
```

Comment:
I would suggest using `assert.compareArray` for these, it's more precise and you'll get better error messages if the test fails.

```suggestion
assert.compareArray(doExec(/𠮷/).toString(), ["𠮷", 0], "Basic exec without v flag");
```
(etc.)

Same suggestion for elsewhere throughout the PR.

You'll need to add `includes: [compareArray.js]` to the front matter for that to work.

Reply:
The assert.compareArray only works for one-dimensional arrays. For the matchAll test, should I just keep my original string comparison?

Reply:
Not ideal (especially the last assertion with empty matches is kind of confusing), but if you want to leave it as is I'm OK with it.

`flatMap` would be nicest, but I'd prefer not to fail the test on engines that haven't implemented `flatMap` yet. So that's out.

One possibility could be to do something like this:
```js
function doMatchAll(regex) {
  const result = RegExp.prototype[Symbol.matchAll].call(regex, text);
  const matches = Array.from(result, m => m[0]);
  const indices = Array.from(result, m => m.index);
  return matches.concat(indices);
}

// ...

assert.compareArray(
  doMatchAll(/𠮷/g),
  ["𠮷", "𠮷", "𠮷", 0, 3, 6],
  "Basic matchAll with g flag"
);

// ...

assert.compareArray(
  doMatchAll(/(?:)/gv),
  ["", "", "", "", "", "", 0, 2, 3, 5, 6, 8],
  "Empty matches with v flag"
);
```

Reply:
Good idea. I can try this in the follow-up PR. BTW, thanks for the feedback, which is very helpful.